README.md

```diff
@@ -87,8 +87,9 @@ is done to make testing easier and to make emails easy to pipe/compose.
 # In your config/config.exs file
 config :my_app, MyApp.Mailer,
   adapter: Bamboo.MandrillAdapter,
-  api_key: "my_api_key"
-
+  api_key: "my_api_key",
+  domain: "my-domain.com" # Set the domain in case you're using Mailgun
```

Comment:
I think this is too specific to just Mailgun. I think instead there should be a comment that says "checks the docs for any adapter specific configuration" or something like that?